src/wazuh_modules/vulnerability_scanner/tests/unit/eventDetailsBuilder_test.cpp

```diff
@@ -809,3 +809,371 @@ TEST_F(EventDetailsBuilderTest, TestSuccessfulOsInserted)
+    {
+        elementOsVersion += ".";
+        elementOsVersion += scanContext->osPatch();
+    }
+    if (!scanContext->osBuild().empty())
+    {
+        elementOsVersion += ".";
+        elementOsVersion += scanContext->osBuild();
+    }
+    EXPECT_STREQ(elementData.at("host").at("os").at("version").get_ref<const std::string&>().c_str(),
+                 elementOsVersion.c_str());
+
+    EXPECT_STREQ(elementData.at("vulnerability").at("category").get_ref<const std::string&>().c_str(), "Packages");
+    EXPECT_STREQ(elementData.at("vulnerability").at("classification").get_ref<const std::string&>().c_str(),
+                 GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->classification()->c_str());
+    EXPECT_STREQ(elementData.at("vulnerability").at("description").get_ref<const std::string&>().c_str(),
+                 GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->description()->c_str());
+    EXPECT_STREQ(elementData.at("vulnerability").at("enumeration").get_ref<const std::string&>().c_str(), "CVE");
+    EXPECT_STREQ(elementData.at("vulnerability").at("id").get_ref<const std::string&>().c_str(), CVEID.c_str());
+    EXPECT_STREQ(elementData.at("vulnerability").at("reference").get_ref<const std::string&>().c_str(),
+                 GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->reference()->c_str());
+    EXPECT_STREQ(elementData.at("vulnerability").at("scanner").at("vendor").get_ref<const std::string&>().c_str(),
+                 "Wazuh");
+    EXPECT_DOUBLE_EQ(
+        elementData.at("vulnerability").at("score").at("base").get_ref<const double&>(),
+        Utils::floatToDoubleRound(GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->scoreBase(), 2));
+    EXPECT_TRUE(elementData.at("wazuh").at("vulnerability").at("under_evaluation"));
+    EXPECT_STREQ(elementData.at("vulnerability").at("score").at("version").get_ref<const std::string&>().c_str(),
+                 GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->scoreVersion()->c_str());
+    EXPECT_STREQ(
+        elementData.at("vulnerability").at("severity").get_ref<const std::string&>().c_str(),
+        Utils::toSentenceCase(GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->severity()->str()).c_str());
+    auto clusterName = PolicyManager::instance().getClusterName();
+    EXPECT_STREQ(elementData.at("wazuh").at("cluster").at("name").get_ref<const std::string&>().c_str(),
+                 clusterName.c_str());
+    EXPECT_STREQ(elementData.at("wazuh").at("schema").at("version").get_ref<const std::string&>().c_str(),
+                 WAZUH_SCHEMA_VERSION);
+    EXPECT_STREQ(elementData.at("vulnerability").at("published_at").get_ref<const std::string&>().c_str(),
+                 GetVulnerabilityDescription(fbBuilder.GetBufferPointer())->datePublished()->c_str());
+    EXPECT_TRUE(elementData.at("vulnerability").at("detected_at").get_ref<const std::string&>() <=
+                Utils::getCurrentISO8601());
+}
```

Comment:
Add missing new line
```suggestion
}

```

Reply:
Fixed!